content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -200,30 +281,32 @@ const result = await prisma.$queryRaw`SELECT * FROM User ${
 
 ### Parameterized queries
 
-As an alternative to tagged templates, `$queryRaw()` and `$executeRaw()` support standard parameterized queries where each variable is represented by a symbol (`?` for mySQL, `$1`, `$2`, and so on for PostgreSQL). The following example uses a MySQL query:
+As an alternative to tagged templates, `$queryRaw` and `$executeRaw` support standard parameterized queries where each variable is represented by a symbol (`?` for mySQL, `$1`, `$2`, and so on for PostgreSQL). The following example uses a MySQL query:
 
 ```ts
 const userName = 'Sarah'
-const email = 'alice@prisma.io'
-const result = await prisma.$queryRaw(
-  'SELECT * FROM User WHERE (name = ? OR email = ?)', // MySQL variables, represented by ?
+const email = 'sarah@prisma.io'
+const result = await prisma.$queryRaw`
```

Comment:
This now belongs under the unsafe category @molebox

Reply:
The whole Parameterized queries section?

Reply:
So moved to SQL injection section?